### PR TITLE
fix(developer): handle merge commits when checking git log date

### PR DESCRIPTION
**Pull request by mcdurdin**
Relates-to: keymanapp/keyboards#3143
Fixes: #12626
Fixes: KEYMAN-DEVELOPER-2AY

@keymanapp-test-bot skip

### CODE REVIEW

File: developer/src/kmc/src/util/getLastGitCommitDate.ts [hunk 1]
```diff
@@ -10,21 +10,53 @@ export const expectedGitDateFormat = /^\d\d\d\d-\d\d-\d\dT\d\d:\d\d:\d\d(\.\d\d\
  */
 export function getLastGitCommitDate(path: string): string {
   // TZ=UTC0 git log -1 --no-merges --date=format:%Y-%m-%dT%H:%M:%SZ --format=%ad
```
Comment (darcywong00) [inline developer/src/kmc/src/util/getLastGitCommitDate.ts:12]: Is the commented format no longer applicable?

**Review by darcywong00 — APPROVED** [review]
lgtm

File: developer/src/kmc/src/util/getLastGitCommitDate.ts (hunk 1 above)
Comment (mcdurdin) [reply developer/src/kmc/src/util/getLastGitCommitDate.ts:12]:
```suggestion
  // git log -1 --no-merges --format=%at
```

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 18.0.136-alpha](https://keyman.com/downloads/releases/alpha/18.0.136)

**Pull request merged**